File: common_ui/src/main/java/cn/woochen/common_ui/adapter/MultiTypeSupport.java
package cn.woochen.common_ui.adapter;

/**
 * 类描述：
 * 创建人：woochen
 * 创建时间：2020/10/28 5:33 PM
 * 修改备注：
 **/
public interface MultiTypeSupport<T> {
    // 根据当前位置或者条目数据返回布局
    int getLayoutId(T item,int position);
}

File: common_ui/src/main/java/cn/woochen/common_ui/adapter/MultiTypeSupport.kt
package cn.woochen.common_ui.adapter

/**
 * 类描述：
 * 创建人：woochen123
 * 创建时间：2017/7/26 22:50
 */
interface MultiTypeSupport<T> {
    // 根据当前位置或者条目数据返回布局
    fun getLayoutId(item: T, position: Int): Int
}
